src/components/users/UserList.js

```diff
@@ -30,7 +30,7 @@ export const UserList = () => {
         let isFriends = false
         return users.filter(user => {
             const friendCheck = filteredFriends.find(f => f.userId === user.id)
-            if ( friendCheck !== undefined) {
+            if (friendCheck !== undefined) {
```

Comment:
What we talked about - friendCheck should be responsible for friend data on its own